### PR TITLE
git-secret 0.1.2 (new formula)

### PR DESCRIPTION
**Pull request by sobolevn**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Earlier it was maintained as a separate `tap`. See:
https://github.com/sobolevn/homebrew-tap/blob/master/Formula/git-secret.rb

### CODE REVIEW

File: Formula/git-secret.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class GitSecret < Formula
+  desc "Bash-tool to store the private data inside a git repo."
+  homepage "https://sobolevn.github.io/git-secret/"
+
```
Comment (DomT4) [inline Formula/git-secret.rb]: Don't need this empty line.

File: Formula/git-secret.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  url "https://github.com/sobolevn/git-secret/archive/v0.1.2.tar.gz"
+  sha256 "96032c1c0b2b161c4df5b8627be8d98a97a92a5d32127c9a5e8686bd64b5a3e7"
+  head "https://github.com/sobolevn/git-secret.git"
+
+  bottle :unneeded
+  depends_on "gnupg" => :recommended
```
Comment (DomT4) [inline Formula/git-secret.rb]: Should be `depends_on :gpg => :recommended`

File: Formula/git-secret.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+class GitSecret < Formula
+  desc "Bash-tool to store the private data inside a git repo."
+  homepage "https://sobolevn.github.io/git-secret/"
+
+  url "https://github.com/sobolevn/git-secret/archive/v0.1.2.tar.gz"
+  sha256 "96032c1c0b2b161c4df5b8627be8d98a97a92a5d32127c9a5e8686bd64b5a3e7"
+  head "https://github.com/sobolevn/git-secret.git"
+
+  bottle :unneeded
```
Comment (DomT4) [inline Formula/git-secret.rb]: You'll need bottles given it has a `make` step.

File: Formula/git-secret.rb [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  depends_on "gnupg" => :recommended
+
+  def install
+    system "make", "build"
+    system "bash", "utils/install.sh", prefix
+  end
+
+  def caveats
```
Comment (DomT4) [inline Formula/git-secret.rb]: Scrap these unless they're Homebrew specific.

File: Formula/git-secret.rb [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    <<-EOS.undent
+      You now have the `git secret` command! Find out how to use it
+      by typing `man git-secret` or visiting 'https://sobolevn.github.io/git-secret/'.
+    EOS
+  end
+
+  test do
+    system "git", "secret", "--version"
```
Comment (DomT4) [inline Formula/git-secret.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by sobolevn** [comment]
Thanks, modified everything. Rebased and updated.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `git-secret 0.1.2 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**